### ISSUE TITLE
Doesn't work on Mac

### ISSUE THREAD

**Issue by jminardi** [Printrbot/firmware-updater#4]
I am running macOS 10.12.3 and have not been able to get the firmware updater to work at all. It just "downloads" forever. I also  tried dragging in a hex file I dl'd from github and it failed as well.

I have a Printerbot Play 1505 with a Printrboard Rev. F5

I noticed people say it doesn't work for them all over the internet, see here (as well as on the printrbot forums):
http://www.instructables.com/id/How-to-Update-Your-Printrbot-Play-1505-Firmware/

**Comment by alberto2000** [comment]
Yes I have the same issue here. It's a bit frustrating, from my perspective Printrbot doesn't seem to maintain their software very well, neither are documentation and links on their website up to date etc. it's all a bit messy... you can use this older updater app which seems works for me: http://www.printrbottalk.com/wiki/index.php
